### ISSUE TITLE
[Suggestion] Prebuilt "alvr-bin" package for AUR

### ISSUE THREAD

**Issue by ClaudeLib** [alvr-org/ALVR#1890]
Strictly low priority, merely a convenience thing. Maybe there are reasons why this isn't a thing, I don't know Linux that well. Most of the software I know provide a separate binary package that doesn't require compiling on end user's side. Would be nice if ALVR had this as well. Thanks.

**Comment by m00nwtchr** [comment]
You can use Chaotic AUR repos (https://aur.chaotic.cx/)

I might add anaj "official" bin package at some point, but only once I have the automation for AUR packages set up

**Comment by ClaudeLib** [comment]
I will check it out, thank you!
